rusk/src/lib/chain/rusk.rs

```diff
@@ -569,6 +576,11 @@ fn reward_slash_and_update_root(
         reward = dusk_value
     );
 
+    let reward_per_quota = generator_extra_reward / (21 * 2);
```

Comment:
Please use config consts to compute the extra credits:

```suggestion
    let validation_quorum = (VALIDATION_COMMITTEE_CREDITS
            * config::SUPERMAJORITY_THRESHOLD)
            .ceil() as usize;
    let validation_extra = VALIDATION_COMMITTEE_CREDITS - validation_quorum;

    let ratification_quorum = (RATIFICATION_COMMITTEE_CREDITS
            * config::SUPERMAJORITY_THRESHOLD)
            .ceil() as usize;
    let ratification_extra = RATIFICATION_COMMITTEE_CREDITS - ratification_quorum;
    let reward_per_quota = generator_extra_reward / (validation_extra + ratification_extra);
```

Reply:
Agreed. These should be actually added into consensus/config.rs and then used here properly.

Reply:
Yes, even better